packages/configuration-validation/package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "@okta/configuration-validation",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Configuration validation support for Okta JavaScript SDKs",
+  "type": "commonjs",
```

Comment:
probably can remove this field, as we support two bundles (cjs, esm) in this package.